File: Practica5/Marco teorico/1. Referencias/1.cpp
#include<iostream>
using namespace std;

int variableEntera = 0;
float variableFlotante = 0;
int& referenciaVariableEntera = variableEntera;
float& referenciaVariableFlotante = variableFlotante;

int main()
{
    cout << referenciaVariableEntera << endl;
    cout << referenciaVariableFlotante << endl;
    return 0;
}

File: Practica5/Marco teorico/1. Referencias/1.java
public class Main {
    static int variableEntera = 0;
    static float variableFlotante = 0;
    static int referenciaVariableEntera = variableEntera;
    static float referenciaVariableFlotante = variableFlotante;

    public static void main(String[] args) {
        System.out.println(referenciaVariableEntera);
        System.out.println(referenciaVariableFlotante);
    }
}
